src/umm_poison.c

```diff
@@ -75,13 +75,13 @@ static bool check_poison_block(umm_block *pblock) {
         void *pc = (void *)pblock->body.data;
```

Comment:
What about just doing this:

`uint8_t *pc = (uint8_t *)pblock->body.data;`

And then leaving everything else alone?

Reply:
Done